src/fixed-footer-layout.css.ts

```diff
@@ -49,6 +48,19 @@ export const container = style({
     },
 });
 
+export const portal = style([
```

Comment:
Given that portals are now rendered with `isolation: isolate`, I had to add this to the FixedFooterLayout portal that is used to render the background (we want it to be rendered under all the other portals)

Reply:
Please include code comments for these special cases. Otherwise, in two weeks you won't remember why we needed a zindex -1 here

Reply:
done